homeassistant/components/nissan_leaf/binary_sensor.py

```diff
@@ -0,0 +1,44 @@
+"""Plugged In Status Support for the Nissan Leaf."""
+import logging
+
+from homeassistant.components.nissan_leaf import (
+    DATA_LEAF, DATA_PLUGGED_IN, LeafEntity)
+
+_LOGGER = logging.getLogger(__name__)
+
+DEPENDENCIES = ['nissan_leaf']
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up of a Nissan Leaf binary sensor."""
+    _LOGGER.debug(
+        "binary_sensor setup_platform, discovery_info=%s", discovery_info)
```

Comment:
Remove this.

Reply:
Fixed with commit https://github.com/filcole/home-assistant/commit/119b2a3b20db6ea4cf05fda29921930c5e27bc86